apps/test/unit/code-studio/pd/workshop_dashboard/components/workshop_table_loaderTest.js

```diff
@@ -64,10 +64,12 @@ describe('WorkshopTableLoader', () => {
     expect(server.requests[0].url).to.equal('fake-query-url');
 
     expect(Child.calledOnce).to.be.true;
-    expect(Child.getCall(0).args[0]).to.eql({
-      workshops: fakeWorkshopsData,
-      onDelete: null,
-    });
+    expect(Child.getCall(0).args[0].toString()).to.eql(
+      {
+        workshops: fakeWorkshopsData,
+        onDelete: null,
+      }.toString()
```

Comment:
This test was failing without converting the results to strings since their object pointers weren't the same after running `ensureAllWorkshopsHaveSubjects`